ygnmi/ygnmi.go

```diff
@@ -333,6 +335,12 @@ func WithDatapointValidator(fn ValidateFn) Option {
 	}
 }
 
+func WithAppendModuleName(append bool) Option {
```

Comment:
Can we add a docstring here please?

Reply:
Added in 483ce8aac7e5e22b874a1da8c8bae9d8fb3d879a.